src/utils.rs

```diff
@@ -140,8 +146,8 @@ CMake suite maintained and supported by Kitware (kitware.com/cmake).\n",
             stdout: String::from("Terraform v0.12.14"),
```

Comment:
Terraform version includes a new line,
```
❯ terraform version | cat -A
Terraform v0.12.28$
```

```suggestion
            stdout: String::from("Terraform v0.12.14\n"),
```